tool/src/org/antlr/v4/codegen/model/chunk/NamedActionChunk.java

```diff
@@ -0,0 +1,14 @@
+package org.antlr.v4.codegen.model.chunk;
+
+import org.antlr.v4.codegen.model.decl.StructDecl;
+
+public abstract class NamedActionChunk extends ActionChunk {
```

Comment:
I'm not sure NamedActionChunk name is the right one.  Looks to me like it is some kind of symbol reference chunk.

Reply:
I'm not sure about this entity naming, probably a more deep understanding is needed. Now it's superclass for `LabelRef`, `LocalRef`, `NonLocalAttrRef`, `RetValueRef`, `SetAttr`, `TokenRef`.

Ok, maybe you are right. I'm renaming it to `SymbolReferenceChunck`, is it ok?

Reply:
Sure maybe shorter like `SymbolRefChunk` to be similar.